app/editor/src/features/admin/cbra/CBRAReport.tsx

```diff
@@ -0,0 +1,79 @@
+import { Button, ButtonVariant } from 'components';
+import { Dropdown, IOptionItem, SelectDate } from 'components/form';
+import { useApiEditor } from 'hooks';
+import React from 'react';
+
+import { defaultValues, reportDurations } from './constants';
+import { ICBRAForm } from './interfaces';
+import * as styled from './styled';
+import { calcDuration } from './utils';
+
+interface ICBRAReportProps extends React.HTMLAttributes<HTMLDivElement> {}
+
+export const CBRAReport: React.FC<ICBRAReportProps> = ({ className }) => {
+  const api = useApiEditor();
```

Comment:
Need to switch to store hooks for all api calls so that we can build in loading.  Will do this next week.